File: ProvedorLogisticaJava/src/main/java/br/inatel/fredlaranjo/dm107/provedorlogistica/dao/UserDao.java
package br.inatel.fredlaranjo.dm107.provedorlogistica.dao;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.SQLException;

public class UserDao {

	private static final String SELECT_USER = "SELECT COUNT(id) FROM users " + "WHERE username = ? AND password = ?;";

	public boolean userExists(String user, String pwd) {
		boolean userExists = false;

		try (Connection connection = ConnectionFactory.getConnection();
				PreparedStatement pstmt = connection.prepareStatement(SELECT_USER)) {
			pstmt.setString(1, user);
			pstmt.setString(2, pwd);

			try (ResultSet rs = pstmt.executeQuery()) {
				int count = 0;

				while (rs.next()) {
					count = rs.getInt(1);
				}

				if (count == 1) {
					userExists = true;
				}
			}
		} catch (SQLException e) {
			userExists = false;
		}

		return userExists;
	}

}

File: ProvedorLogisticaPHP/src/public/userdao.php
<?php
class UserDao {
    private $db;
    public function __construct() {
        $this->db = Connection::getConnection();
    }

    public function userExists($user, $pwd) {
        $userCount = $this->db->user()->where("user = ?", $user)->where("pwd = ?", $pwd)->count("*");

        if ($userCount == 0) {
            return false;
        }

        return true;
    }

     public function getUsers() {
        $users = array();

        foreach ($this->db->user() as $user) {
            $users[$user["user"]] = $user["pwd"];
        }

        return $users;
    }
}
?>